src/constants_and_utils/utils.js

```diff
@@ -17,8 +19,46 @@ function waitForElementToBeClickable(htmlElement, timeoutInMilliseconds, errorMe
   browser.wait(EC.elementToBeClickable(htmlElement), timeoutInMilliseconds, errorMessage);
 }
 
+function protractorHelperWarning() {
+  return colors.yellow("Protractor-helper warning: ");
+}
+
+function readMoreDoc() {
+  return colors.cyan(" [Read more on www.npmjs.com/package/protractor-helper#preparation-to-next-major-version]");
+}
+
+function obsoleteFunction(functionName, text = "") {
+  console.warn(
+    protractorHelperWarning() +
+      "Function '" +
+      functionName +
+      "' will be deprecated in version 4.0.0! " +
+      text +
+      readMoreDoc()
+  );
+}
+
+function replaceObsoleteFunction(oldFunctionName, newFunctionName) {
+  obsoleteFunction(oldFunctionName, "Please use the new '" + newFunctionName + "' function instead!");
+}
+
+function warnRemoveErrorMessage(functionName, errorMessage, defaultMessage) {
+  if (errorMessage != defaultMessage) {
+    console.warn(
+      protractorHelperWarning() +
```

Comment:
Same here (https://github.com/wlsf82/protractor-helper/pull/43/commits/fbc07a52843f0628997cfc6a584a92c1c521d8a9#r289022891).

Reply:
I will do it.

Reply:
Don't reply `I will do it.' in all comments, please. Just make the change, commit, and push.